src/main/java/com/example/bcsd/domain/Article.java

```diff
@@ -0,0 +1,60 @@
+package com.example.bcsd.domain;
+
+import java.time.LocalDateTime;
+
+public class Article {
+    private final Long id;
+    private final Long authorId;
+    private Long boardId;
+    private String title;
+    private String content;
+    private final LocalDateTime createdDate;
+    private LocalDateTime modifiedDate;
+
+    public Article(Long id, Long authorId, Long boardId,
+                   String title, String content,
+                   LocalDateTime createdDate, LocalDateTime modifiedDate) {
+        this.id = id;
+        this.authorId = authorId;
+        this.boardId = boardId;
+        this.title = title;
+        this.content = content;
+        this.createdDate = createdDate;
+        this.modifiedDate = modifiedDate;
+    }
```

Comment:
```suggestion
    public Article(
        Long id, 
        Long authorId, 
        Long boardId,
        String title, 
        String content,
        LocalDateTime createdDate, 
        LocalDateTime modifiedDate
      ) {
        this.id = id;
        this.authorId = authorId;
        this.boardId = boardId;
        this.title = title;
        this.content = content;
        this.createdDate = createdDate;
        this.modifiedDate = modifiedDate;
    }
```

이 방식이 가독성이 좀 더 좋을 것 같아요